Makefile

```diff
@@ -123,16 +123,22 @@ pyenv:
 		sed -i -e "s|^dev_sources = $(PWD)$$|dev_sources = $(PWD)/python|" ~/.azure/config
```

Comment:
sorry for the noise - needed a pr w/o any comments/reviews and am testing something

Reply:
2